README.md

```diff
@@ -58,96 +58,95 @@ pip install jupyterhub-ltiauthenticator
     is available to help. Your LMS provider might also implement custom keys
     you can use.
 
-5.  Pick a name for edX to call your JupyterHub server. Then, along with the two random strings you generated in step 4, paste them together to create an _LTI Passport String_ in the following format:
+- **consumers**: the value for the LTI 1.1 configuration consumer key and shared secret created in Step 1 of [the common configuration settings section](#common-configuration-settings).
+- **username_key**: the value for the LTI 1.1 configuration XML description.
 
-    ```
-    your-hub-name:client-key:client-secret
-    ```
+2. By default, the user's name will be the `custom_canvas_user_id` passed in by canvas. If you would like to use some other bit of information from the LTI request, you can pick what should be used as the user id.
 
-    `your-hub-name` can be anything, but you'll be using it throughout edX to refer to your hub, so make it something meaningful and unique.
+```python
+# Set the user's email as their user id
+c.LTIAuthenticator.username_key = 'lis_person_contact_email_primary'
+```
 
-    Here's an example:
+A [partial list of keys in an LTI request](https://www.edu-apps.org/code.html#params)
+is available to help. Your LMS provider might also implement custom keys
+you can use.
 
-    ```
-    stat100-jupyterhub:fca69fa2deda7d45dfdfa85f288d59b4b5f3d22bfca2ba891187fe5c551705a5:8ce3caca82f467f0896b92d548dfbddaca86edfcaaba8c9d6777dfae4d2e1db9
-    ```
+3. If you are running **Jupyterhub within a Kubernetes Cluster**, deployed using helm, you need to supply the client key & secret via the chart yaml configuration.
 
-    Then [add the Passport String](http://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/lti_component.html#adding-an-lti-passport-to-the-course-configuration)
-    to EdX. Remember to save your changes when done!
+config.yaml:
 
-6.  Configure JupyterHub to accept LTI Launch requests from EdX. You do this by
-    supplying JupyterHub with the client key & secret generated in step 3 (you don't need the hub name from step 4).
+```yaml
+    auth:
+      type: "lti"
+      lti:
+          consumers: {
+              "client-key"": "client-secret"
+              }
+```
```

Comment:
Indentation inconsistencies and double an uneven amount of `"` used making it invalid yaml.